Jellyfin.Server/Program.cs

```diff
@@ -378,7 +378,7 @@ public static void PerformStaticInitialization()
                 .ConfigureServices(services =>
                 {
                     // Merge the external ServiceCollection into ASP.NET DI
-                    services.TryAdd(serviceCollection);
+                    services.Add(serviceCollection);
```

Comment:
This change is necessary because `TryAdd` will only allow 1 implementation per interface.